C#/EfcDataAccess/SensorEfcDao.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using Application.DaoInterfaces;
+using Domain.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+
+namespace EfcDataAccess;
+
+public class SensorEfcDao : ISensorDao
+{
+
+    private readonly HospitalContext context;
+
+    public SensorEfcDao(HospitalContext context)
+    {
+        this.context = context;
+    }
+    
+    public async Task<Sensor> CreateAsync(Sensor sensor)
+    {
+        EntityEntry<Sensor> newSensor = await context.Sensors.AddAsync(sensor);
+        await context.SaveChangesAsync();
+        return newSensor.Entity;
+    }
+
+    public async Task<Sensor?> GetById(int id)
+    {
+        Sensor? sensor = await context.Sensors.FindAsync(id);
+        return sensor;    }
+
+    public async Task<IEnumerable<SensorValue>> GetSensorsValuesAsync(int? roomId)
```

Comment:
Is this method for the log or the current sensor value? The IEnumerable lets me believe that it's the log, but then there is no method to get the current sensor value...and we have an endpoint for that, therefore I need that method.
And to get the log of a sensor and the current value, we need the sensor's id. We can't just get from every sensor without knowing which sensor we are taking the values of.
Basically, I need you to add the sensorId and make a separate method to get only one SensorValue (the last one that was saved in the database / the current one).